app/templates/gentelella/guest/event/schedule.html

```diff
@@ -151,15 +151,16 @@ <h1 style="font-weight: 300; font-size: 24px">
                     $('.overlay').addClass('visible');
                     $('.btn-pop-out').removeClass('fa-expand').addClass('fa-compress');
                     var width = $("#timeline").width() - $('.timeunits.x1').width();
-                    $('.microlocation-container').css("width", width + "px");
+                    $('.microlocation-container').css("max-width", width + "px");
                     $('.session.scheduled').popover('hide');
                 }
                 function close(){
                     $scheduler.removeClass('scheduler-pop');
                     $body.removeClass('noscroll');
                     $('.overlay').removeClass('visible');
                     $('.btn-pop-out').addClass('fa-expand').removeClass('fa-compress');
-                    $('.microlocation-container').css("width", "fit-content");
```

Comment:
@SaptakS Yes. I have tested that. Its not leaving extra space on the right side because now microlocation-container has only max-width property hence will wrap. the previous screenshot was before commit. 

![screenshot from 2017-05-17 11-27-14](https://cloud.githubusercontent.com/assets/13533840/26140439/78aaf7fe-3af4-11e7-8469-7e93ac9582f7.png)